net-libs/libmodbus/libmodbus-3.0.8.recipe

```diff
@@ -25,15 +25,18 @@ CHECKSUM_SHA256="0e236c3d4fe0a7a3f7c6292a3d1183c584b03bb06154e0b95e71bc763559ac5
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%.*}"
+
 PROVIDES="
-	libmodbus$secondaryArchSuffix = $portVersion
+	lib:libmodbus$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
-	libmodbus${secondaryArchSuffix}_devel = $portVersion
+	devel:libmodbus${secondaryArchSuffix}_devel = $portVersion
```

Comment:
Keep this too. Both required.

Reply:
Let me correct myself: this line should be changed to:
```
devel:libmodbus$secondaryArchSuffix = $portVersion
```

Reply:
ok done